src/mixins/object_geometry.mixin.js

```diff
@@ -431,7 +431,7 @@
 
     calcLineCoords: function() {
       var vpt = this.getViewportTransform(),
-          padding = this.padding, angle = degreesToRadians(this.angle),
+          padding = this.padding, angle = degreesToRadians(this.getTotalAngle()),
```

Comment:
Not 100% sure about this, 90%

Reply:
We need to test this with skewed objects.
I'm not sure is right.

Reply:
skew doesn't work in oCoords as well

Reply:
at canvas level it works, but not in group currently if you have  skewY ( or skewX + skewY )

Reply:
I tried fixing oCoords to include skewing but it's above my understanding